### DISCUSSION TITLE
v2 upload audio file

### DISCUSSION THREAD

**Discussion by yablochnyi** [filamentphp/filament#7952] [Help] [answered]
Hello, when developing locally, I upload a file and in the admin panel I can listen to it
![Снимок экрана 2023-08-21 115446](https://github.com/filamentphp/filament/assets/75489338/9c821b5b-7d8c-41f5-aca2-eec08b648c91)
I can't listen to the file on the server
![1](https://github.com/filamentphp/filament/assets/75489338/f29c3d23-06ab-49d4-a1a8-22ba8d60a16e)

**Comment by pboivin** [comment, marked as answer]
Do you have any errors in the browser console?

**Reply by yablochnyi** [reply]
no

**Comment by yablochnyi** [comment]
Help me, I'm still looking for a solution

**Reply by danharrin** [reply]
Your question lacks the detail required for anyone to help

**Reply by yablochnyi** [reply]
I don't know what details I can provide. I have an audio file in vaw format and I want to be able to listen to it in the admin panel

**Reply by yablochnyi** [reply]
?
